### DISCUSSION TITLE
UCUM openEHR certification

### DISCUSSION THREAD

**Discussion by BarbaannaBPM** [ucum-org/ucum#297] [Q&A]
Do we have any insights on whether openEHR is planning to certify UCUM?

**Comment by chgessner** [comment]
I don't know and IMHO such certification would not be something that the UCUM board would deal with. I understand that you are referring to this publication: https://specifications.openehr.org/releases/CNF/latest/guide.html

That said, I want to remind the community that we have a discussion still ongoing that relates to "conformance criteria", see here: https://github.com/orgs/ucum-org/discussions/250
